### ISSUE TITLE
Pretty strap repo url

### ISSUE THREAD

**Issue by fvumbaca** [azohra/strapped#17]
*We should make the strapped repo URL more pretty!*

From:
``` YAML
strapped:
  repo: https://raw.githubusercontent.com/azohra/strapped/master/straps
```
to:
``` YAML
strapped:
  repo: repo.strapped.sh
```

This can be done by adding an HTTP redirect from `repo.strapped.sh` to `https://raw.githubusercontent.com/azohra/strapped/master/straps`. The code can support this by adding the `-L` follow-redirect flag to the curl call that pulls the referenced strap repo.

**Comment by MatthewNielsen27** [comment]
@tk8817 could you make these changes?

**Comment by azohra** [comment]
It is doable thanks to Netlify in this case, we will have them serve the page and do a 301 redirect.  If it was pure DNS, we would not be able to do the path.

**Comment by azohra** [comment]
This was anything but easy lol.

We actually needed to do full domain rewriting to support this:


This is easy (DNS doable)
https://repo.strapped.sh > https://some.other.domain

This is harder (301 redirect)
https://repo.strapped.sh > https://raw.githubusercontent.com/azohra/strapped.sh/master/straps

This is hardest (301 redirect + path rewrite) 
https://repo.strapped.sh/* > https://raw.githubusercontent.com/azohra/strapped.sh/master/straps/*

example of why we need it: 
https://repo.strapped.sh/brew/spec.yml > https://raw.githubusercontent.com/azohra/strapped.sh/master/straps/brew/spec.yml

**Comment by azohra** [comment]
resolved with 0.2.0

**Comment by MatthewNielsen27** [comment]
Thank you papa Justin <3
